src/game/multiplayer/communication_outcome.c

```diff
@@ -218,17 +345,26 @@ static void sub_805BC40(void)
         u8 outcome = outcomeScreen->unk203;
         TasksDestroyAll();
         PAUSE_BACKGROUNDS_QUEUE();
-        gUnknown_03005390 = 0;
+        SA2_LABEL(gUnknown_03005390) = 0;
         PAUSE_GRAPHICS_QUEUE();
         if (outcome == OUTCOME_CONNECTION_SUCCESS) {
+
+#if (GAME == GAME_SA1)
+            CreateMultiplayerResultsScreen(0);
+#elif (GAME == GAME_SA2)
             CreateCharacterSelectionScreen(gSelectedCharacter, gMultiplayerUnlockedCharacters & CHARACTER_BIT(CHARACTER_AMY));
+#endif
         } else {
             gMultiSioEnabled = FALSE;
             MultiSioStop();
             MultiSioInit(0);
 
             gTilemapsRef = (Tilemap **)gTilemaps;
+#if (GAME == GAME_SA1)
+            CreateMainMenu(1);
```

Comment:
We could potentially align the names of these? They do the same thing right? The sa2 one is more literal, but in theory it could be called "CreateMainMenu" if that helps. I've not looked at how the intro and menu are linked in SA1

Reply:
`CreateMainMenu(1)` vs `CreateTitleScreenAndSkipIntro();`

Are these not the same functionality?

Reply:
No, those two also aren't.

`CreateMainMenu` is about the main menu, not the "Press Start" titlescreen, no matter which parameters are set.
`CreateTitleScreenAndSkipIntro` opens the titlescreen itself, and is called when exiting the Time Attack, or MP modes, etc.